src/frontend/Komponenter/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeBarnetilsyn.tsx

```diff
@@ -34,6 +34,7 @@ export const InnvilgeBarnetilsyn: FC<{
 
     const hentVedtakshistorikk = useCallback(
         (revurderesFra: string) => {
+            settVedtakshistorikk(undefined);
```

Comment:
Tror ikke jeg helt forstod hvorfor denne linjen ble lagt til

Reply:
For å få en rendering uten vedtakshistorikk, for å "nullstille" `Vedtaksform`
Lite likt som når vi bruker `settVedtakshistorikk(byggTomRessurs)`